nf_core/lint/files_exist.py

```diff
@@ -34,15 +34,17 @@ def files_exist(self):
         [LICENSE, LICENSE.md, LICENCE, LICENCE.md]  # NB: British / American spelling
         assets/email_template.html
         assets/email_template.txt
-        assets/nf-core-PIPELINE_logo.png
```

Comment:
Maybe also add a test that the old filename must _not_ be there, to make sure that people clean up the old orphaned filenames..